bundle/tests/scorecard/kuttl/network-policy/01-assert.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: network-policy-rc
+spec:
+  ingress:
+    - from:
+        - namespaceSelector: {}
```

Comment:
Is it necessary for the port(s) to be specified for Route/Ingress to work? If not, we should only specify ports when fromLabels is specified (to allow traffic from certain pods).
Also, is traffic restricted for all namespaces, including the CR namespace?

Reply:
> Also, is traffic restricted for all namespaces, including the CR namespace?

Yes. With namespaceSelector: {} it should apply to all ingress traffic from any namespace.

> Is it necessary for the port(s) to be specified for Route/Ingress to work? If not, we should only specify ports when fromLabels is specified (to allow traffic from certain pods).

Yes, without the ports specified, external traffic would be blocked.